src/cljs/replumb/repl.cljs

```diff
@@ -827,7 +853,14 @@
   file is found). It is mutually exclusive with :load-fn! and will be
   ignored in case both are present
 
-  * :src-paths - a vector of paths containing source files
+  * `:write-file-fn!` a synchronous 2-arity function that accepts a file-path
+  and data to write.
+
+  * `:src-paths` - a vector of paths containing source files
+  * `:cache-path` - if present, will indicate the path of the cached files
+  * `:search-cache-in-src-paths?` - if true, will look for cached files in
```

Comment:
this param name is really long, we could have a unique map holding all the cache params:
```
{:cache {:enable? true/false
              :path ...}}
```

Or even more compact:
```
{:cache {:path ...}}
```
where the mere presence of `:cache` always enables it